src/Timeline.js

```diff
@@ -179,14 +175,44 @@ export default class Timeline extends Axis {
     this._brushGroup.selectAll(".overlay")
       .attr("cursor", this._brushing ? "crosshair" : "pointer");
 
-    this._brushGroup.selectAll(".selection")
+    const yTransform = this._height / 2 - this._buttonHeight / 2;
+
+    const brushSelection = this._brushGroup.selectAll(".selection")
       .call(attrize, this._selectionConfig)
       .attr("height", timelineHeight);
 
-    this._brushGroup.selectAll(".handle")
+    const brushHandle = this._brushGroup.selectAll(".handle")
       .call(attrize, this._handleConfig)
       .attr("height", timelineHeight + this._handleSize);
 
+    if (this._buttonBehaviorCurrent === "buttons") {
+      this._brushGroup.selectAll(".handle")
+      .attr("height", this._buttonHeight);
```

Comment:
you're setting the height for `".handle"` twice, move this out of the if block and use a ternary